src/keylessBackup/KeylessBackupPhoneCodeInput.tsx

```diff
@@ -77,6 +126,33 @@ const styles = StyleSheet.create({
     color: colors.primary,
     ...typeScale.labelSemiBoldMedium,
   },
+  bottomSheetTitle: {
+    ...typeScale.titleSmall,
+    textAlign: 'center',
+  },
+  bottomSheetView: {
+    paddingHorizontal: 24,
+    display: 'flex',
+    flexDirection: 'column',
+  },
+  bottomSheetBody: {
+    ...typeScale.bodyMedium,
+    textAlign: 'center',
+  },
+  buttonContainer: {
+    flexDirection: 'row',
+    marginTop: 37,
+    marginBottom: 9,
```

Comment:
These two magic numbers (37 and 9) are copied over from styling from an old deprecated component, it didn't seem like there was a constant with these values (saw them used like this in other places) so not sure if this is okay or something else should be used?